main.c

```diff
@@ -973,15 +1011,27 @@ int main(int argc, char *argv[]) {
 
 	struct wl_registry *registry = wl_display_get_registry(state.display);
 	wl_registry_add_listener(registry, &registry_listener, NULL);
+	wl_display_dispatch(state.display);
```

Comment:
I don't think we need a dispatch here -- `wl_display_roundtrip` takes care of it already.

Reply:
Removing gives me an empty seat list. I think roundtrip waits until the server processes everything while dispatch processes the incoming events.

Reply:
We need two roundtrips: one to retrieve the list of globals, one to retrieve the seat names.

Reply:
Ah gotcha. That does it